components/AgeCard/AgeCard.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+
+const AgeCardContainerWrapper = styled.div`
+  height: 240px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 4px;
+  border: 1px solid gray;
+`;
+
+const AgeCardContainer = styled.div`
+  position: relative;
+  text-align: center;
+  height: 100%;
+  width: 100%;
+`;
+
+const RemainingDiv = styled.div`
+  height: ${({ remainingHeight }) => remainingHeight};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0.3;
+`;
+
+const AgeButtonBadgeContainer = styled.div`
+  position: absolute;
+  z-index: 10;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const AgeButtonBadge = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 16px;
+`;
+
+const AgeButton = styled.button`
+  background-color: transparent;
+  border: 1px solid gray;
+  margin: 4px;
+  cursor: pointer;
+  user-select: none;
+`;
+
+const AgeValue = styled.p`
+  user-select: none;
+`;
+
+const UsedDiv = styled.div`
+  height: ${({ customHeightAge }) => customHeightAge};
```

Comment:
what does this do? never seen that before?

Reply:
i had to use interpolation here as I wanted the height to be the age of the developer. 

let customHeightAge = `${age}%`; <- defined below in the code, uses the age-state converted to % and then is used by the UsedDiv